core/src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/slick/SubmissionComponent.scala

```diff
@@ -432,6 +435,12 @@ trait SubmissionComponent {
       query.result.map(_.toMap)
     }
 
+    def getEmptyOutputParam(submissionId: UUID): ReadAction[Boolean] = {
+      val query = submissionQuery.filter(_.id === submissionId).map(_.ignoreEmptyOutputs)
+
+      query.result.head
+    }
```

Comment:
for older submissions that have a null for the `IGNORE_EMPTY_OUTPUTS` column, I _think_ this will throw an error trying to cast null into Boolean. Even if it doesn't, consider using the `uniqueResult` helper function plus a `getOrElse` to set the default:

```scala
    def getEmptyOutputParam(submissionId: UUID): ReadAction[Boolean] =
      uniqueResult[Boolean](submissionQuery.filter(_.id === submissionId).map(_.ignoreEmptyOutputs))
        .map(_.getOrElse(false))
```
or, return an `Option[Boolean]` and let calling code figure out what to do if it returns null/None:
```scala
    def getEmptyOutputParam(submissionId: UUID): ReadAction[Option[Boolean]] =
      uniqueResult[Boolean](submissionQuery.filter(_.id === submissionId).map(_.ignoreEmptyOutputs))
```

Reply:
Makes sense! Using uniqueResult